configure

```diff
@@ -810,6 +834,12 @@ def configure_node(o):
     o['variables']['node_target_type'] = 'static_library'
 
   o['variables']['node_no_browser_globals'] = b(options.no_browser_globals)
+  o['variables']['node_shared'] = b(options.shared)
+  o['variables']['node_use_v8_platform'] = b(not options.no_v8_platform)
+  o['variables']['node_use_bundled_v8'] = b(not options.no_bundled_v8)
+  
+  if flavor == 'linux' and options.shared:
+    o['variables']['node_module_ver'] = getmoduleversion.get_version()
```

Comment:
I'd be okay with exporting this unconditionally.  It's going to end up in `process.config.variables` where it might be useful (for example) for people building add-ons.  I'd call it `node_module_version` though.